functions/exercise16/script.js

```diff
@@ -1 +1,11 @@
-"use strict";
\ No newline at end of file
+"use strict";
+
+function getUniqueCharactersFromString(string) {
+    var uniqueCharacters = "";
+    for (var i = 0; i < string.length; i++) {
+        if (uniqueCharacters.indexOf(string.charAt(i)) == -1) {
```

Comment:
You refer to current character in two ways:
string.charAt(i)
string[i]
Maybe it would be better to do it only once in a standarised way?